http2-tests/src/test/scala/org/apache/pekko/http/impl/engine/http2/H2SpecIntegrationSpec.scala

```diff
@@ -306,6 +306,7 @@ class H2SpecIntegrationSpec extends PekkoFreeSpec(
         executable,
         "-k", "-t",
         "-p", port.toString,
+        "-o", "9",
```

Comment:
See https://github.com/summerwind/h2spec?tab=readme-ov-file#usage for reference. The default value is 2 seconds, I got to 9 with manual testing by incrememting 1 each time.